invenio_app_ils/config.py

```diff
@@ -373,14 +372,14 @@ def _(x):
         },
         record_serializers={
             "application/json": (
-                "invenio_app_ils.records.serializers:json_v1_response"
+                "invenio_app_ils.literature.serializers:json_v1_response"
```

Comment:
This works for sure, but I am wondering if we should split in different serializers at some point, even if for now the code is the same.